package.json

```diff
@@ -897,6 +898,77 @@
         "uiTheme": "vs",
         "path": "./themes/theme-psise/theme.json"
       }
+    ],
+    "walkthroughs": [
+      {
+        "id": "PowerShell",
+        "title": "Get Started with PowerShell",
+        "description": "Set up your environment and see what the PowerShell Extension in Visual Studio Code can help you achieve!",
+        "steps": [
+          {
+            "id": "version",
+            "title": "Choose a version of PowerShell",
+            "media": {
+              "markdown": "media/walkthrough.md"
+            },
```

Comment:
Note: this empty media key is annoyingly required for the walkthrough to show up, even though it does nothing.

Reply:
Can you open an issue about this?

Reply:
I'm _sure_ it's known already!